manim/mobject/types/point_cloud_mobject.py

```diff
@@ -313,6 +373,15 @@ def generate_points(self):
             ]
         )
 
+    def set_point_width_uniform(self):
+        if config["renderer"] == "opengl":
```

Comment:
It'd be better if this check for opengl was under the `__init__` instead of being in the method + same comment about reducing redundnacy from the `__init_points__` comment.

Reply:
> It'd be better if this check for opengl was under the `__init__` instead of being in the method + same comment about reducing redundnacy from the `__init_points__` comment.

Good point will move this if we don't split separate opengl implementation